### ISSUE TITLE
:wheelchair: a11y(accordion): accordion not accessible

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1419]
### Description of this issue

* Accordion items are not accessible via keyboard. 
* Accordions are missing native properties

<img width="1341" alt="accordion" src="https://github.com/baloise/design-system/assets/172396366/c04b722a-b5b8-4fe3-b81c-19d1706c349a">

### How to fix it

* Screenreader: Accordion doesn't announce whether it is expanded or closed. Maybe aria-expanded is missing.
* Accordion trigger button is read as button twice by NVDA/Windows. Maybe because of button inside of a button. Please make accordion trigger (v-icon) aria-hidden
* Please implement aria-labelledby=bal-accordion-summary at bal-accordion-details. This way hopefully the role=region will be labelled by accordion trigger name (the title of the accordion to open)

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

aria-expanded: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by carolabes** [comment]
Screenreader won't read if the accordion icon is expanded or reduced.

**Comment by GinaBiondo** [comment]
This is related to the accordion variant of the bal-list

**Comment by carolabes** [comment]
FYI: Updated Ticket description
